### PR TITLE
FB - autorenew domain name

### PR DESCRIPTION
**Pull request by fabienOVH**
Waiting for the review of Bérénice Despres and Antoine (not before 26 august)

### CODE REVIEW

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: 'Comment renouveler ses noms de domaine OVHcloud'
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
title: 'Comment renouveler mes noms de domaine OVHcloud'
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+## En pratique
+
+### Pourquoi renouveler son nom de domaine ?
+
+Renouveler correctement votre nom de domaine est crucial pour maintenir votre visibilité en ligne et préserver votre image de marque, tout en évitant des frais supplémentaires.
+
+Le nom de domaine est l'adresse permettant à vos contacts de se rendre sur votre site internet et d'échanger avec vous par e-mail. Si votre nom de domaine n'est pas renouvelé, votre site web et vos adresses e-mails ne seront plus accessibles. Un site web non-accessible pendant quelques temps peut avoir des impacts sur votre visibilité web et sur votre référencement naturel.
+
+Dans le cas où vous n'avez pas renouvelé votre nom de domaine avant la date d'anniversaire de ce dernier alors que vous souhaitiez tout de même le conserver, des frais additionnels peuvent être associés.
+En effet, une fois le nom de domaine supprimé, le renouvellement n'est plus possible et vous devez  restaurer votre nom de domaine. Pour que la restauration puisse être effective, des frais de restauration seront appliqués.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
En effet, une fois le nom de domaine supprimé, le renouvellement n'est plus possible et vous devez alors restaurer votre nom de domaine. Pour que la restauration puisse être effective, des frais de restauration seront appliqués. 
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Dans le cas où vous n'avez pas renouvelé votre nom de domaine avant la date d'anniversaire de ce dernier alors que vous souhaitiez tout de même le conserver, des frais additionnels peuvent être associés.
+En effet, une fois le nom de domaine supprimé, le renouvellement n'est plus possible et vous devez  restaurer votre nom de domaine. Pour que la restauration puisse être effective, des frais de restauration seront appliqués. 
+
+Enfin, si vous ne souhaitez pas conserver votre nom de domaine, il sera automatiquement disponible sur le marché et pourra de nouveau être enregistré par quelqu'un d'autre.
+Si le nom de domaine comporte votre nom de marque, soyez vigilent. Une personne malveillante pourrait utiliser votre ancien nom de domaine pour du phishing ou des arnaques en se faisant passer pour votre entreprise.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
Si le nom de domaine comporte votre nom de marque, soyez vigilant. Une personne malveillante pourrait utiliser votre ancien nom de domaine pour du phishing ou des arnaques en se faisant passer pour votre entreprise.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+Si vous souhaitez conserver votre nom de domaine, nous vous conseillons l'enregistrement et le renouvellement sur plusieurs années, ou l'activation du renouvellement automatique. 
+Si le renouvellement automatique est activé, votre moyen de paiement doit être valide pour que le prélèvement puisse être effectué.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
Si le renouvellement automatique est activé, votre moyen de paiement doit être valide afin que le prélèvement automatique puisse être effectué.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+Si vous choisissez le renouvellement manuel, vous recevrez plusieurs relances avant échéance, contenant un lien permettant de renouveler en ligne le service arrivant à expiration.
+Vous pourrez aussi simplement effectuer le règlement depuis votre espace client.
+
+**Attention**, si vous n'effectuez pas de paiement pour le service en renouvellement manuel, vous risquez de perdre votre nom de domaine ou de régler des frais de restauration (si le nom de domaine est suspendu).
+En revanche, si vous ne souhaitez pas conserver ce service, il n'est pas nécessaire d'en demander la résiliation.
+
+### Changer le mode de renouvellement de mes noms de domaine
+
+Dans votre espace client, cliquez sur votre nom en haut à droite de votre écran puis sur `Produits et services`{.action}. Dans le tableau qui s'affiche, identifiez le ou les noms de domaine dont vous souhaitez changer le mode de renouvellement.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
Dans votre [espace client](/links/manager), cliquez sur votre nom en haut à droite de votre écran puis sur `Produits et services`{.action}. Dans le tableau qui s'affiche, identifiez le ou les noms de domaine dont vous souhaitez changer le mode de renouvellement.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+#### Pour un seul nom de domaine
+
+Effectuez les action suivantes :
+
+- Dans le tableau, identifiez le nom de domaine dont vous voulez changer le mode de renouvellement
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md:82]: Il faudrait un point à la fin de chacune de ces phrases.

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+- Sélectionnez `Activer le paiement automatique`{.action} ou `Désactiver le paiement automatique`{.action}
+
+### Renouveler mon nom de domaine avant la date d'échéance
+
+Effectuez les action suivantes :
+
+- Dans le tableau, identifiez le nom de domaine que vous souhaitez renouveler
+- Cliquez à droite de la ligne sur le bouton `...`{.action}
+- Cliquez sur l'action `Anticiper le paiement`{.action}
+- Après avoir été redirigé vers une interface de paiement en ligne, suivez les instructions
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
- Vous êtes alors redirigé vers une interface de paiement en ligne, suivez les instructions fournies.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+Vous pouvez renouveler le service à tout moment avant son expiration, ainsi que choisir la durée de renouvellement.
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
Vous pouvez renouveler le service à tout moment avant son expiration et également choisir la durée de renouvellement.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+>> Pendant la période de grâce, vous pourrez le renouveler via l'espace client, en suivant les étapes de ce guide.
+>> Pendant la période de rédemption, le domaine n'est plus géré par OVHcloud. Son renouvellement via votre espace client occasionnera des frais supplémentaires (frais de restauration, qui varient en fonction de l'extension. Voir les tarifs en cliquant sur l'extension concernée [ici](https://www.ovhcloud.com/fr/domains/tld/){.external}.).
+>> Au-delà de la période de rédemption, le nom de domaine entre dans une phase de suppression pouvant s'étaler sur plusieurs jours, avant de redevenir disponible dans le domaine public. Si vous souhaitez alors en récupérer l'usage, nous vous conseillons d'utiliser l'outil [Whois d'un nom de domaine](https://www.ovhcloud.com/fr/domains/whois/) pour vérifier sa disponibilité régulièrement.
+> Je ne souhaite pas renouveler mon nom de domaine mais le transférer vers un autre bureau d'entregistrement que OVHcloud. Comment faire ?
```
Comment (Y0Coss) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
> Je ne souhaite pas renouveler mon nom de domaine mais le transférer vers un autre bureau d'enregistrement que OVHcloud. Comment faire ?
```

File: pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Comment renouveler mes services OVHcloud
 excerpt: Découvrez comment gérer vos services et leur renouvellement dans votre espace client
-updated: 2024-01-26
+updated: 2024-08-19
```
Comment (Kazarus4) [inline pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md]:
```suggestion
updated: 2024-11-12
```

File: pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Comment renouveler mes services OVHcloud
```
Comment (Kazarus4) [inline pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md]:
```suggestion
title: "Comment renouveler mes services OVHcloud"
```

File: pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
...
 excerpt: Découvrez comment gérer vos services et leur renouvellement dans votre espace client
```
Comment (Kazarus4) [inline pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal/guide.fr-fr.md]:
```suggestion
excerpt: "Découvrez comment gérer vos services et leur renouvellement dans votre espace client"
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,161 @@
+---
+title: 'Comment renouveler mes noms de domaine OVHcloud'
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
title: "renouveler mes noms de domaine OVHcloud"
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+excerpt: 'Découvrez pourquoi et comment renouveler vos noms de domaine OVHcloud'
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
excerpt: "Découvrez pourquoi et comment renouveler vos noms de domaine OVHcloud"
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+updated: 2024-10-17
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
updated: 2024-11-12
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 16]
```diff
@@ -0,0 +1,161 @@
...
+- Renouveler un nom de domaine avant la date d'échéance
+- Résilier un nom de domaine
+- Accéder à la facture du renouvellement du nom de domaine
+
+**Découvrez pourquoi et comment renouveler vos noms de domaine OVHcloud**
+
+## Prérequis
+
+- Disposer d'un nom de domaine enregistré chez OVHcloud.
+- Être connecté à votre [espace client OVHcloud](/links/manager){.external}, partie `Web Cloud`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
- Être connecté à votre [espace client OVHcloud](/links/manager), partie `Web Cloud`{.action}.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 17]
```diff
@@ -0,0 +1,161 @@
...
+>> Il est possible d’être automatiquement remboursé jusqu’à 10 jours avant la date de renouvellement effective du nom de domaine. Pour cela, résiliez le nom de domaine.
+
+
+## Aller plus loin
+
+Ce guide n'a pas répondu à votre question ? Consultez notre [FAQ sur la facturation et le paiement](/pages/account_and_service_management/managing_billing_payments_and_services/faq-billing)
+
+[Comment renouveler mes services OVHcloud](/pages/account_and_service_management/managing_billing_payments_and_services/how_to_use_automatic_renewal)
+
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com/>.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
Échangez avec notre [communauté d'utilisateurs](/links/community).
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 18]
```diff
@@ -0,0 +1,161 @@
...
+>> Pendant la période de grâce, vous pourrez le renouveler via l'espace client, en suivant les étapes de ce guide.
+>> Pendant la période de rédemption, le domaine n'est plus géré par OVHcloud. Son renouvellement via votre espace client occasionnera des frais supplémentaires (frais de restauration, qui varient en fonction de l'extension. Voir les tarifs en cliquant sur l'extension concernée [ici](https://www.ovhcloud.com/fr/domains/tld/){.external}.).
+>> Au-delà de la période de rédemption, le nom de domaine entre dans une phase de suppression pouvant s'étaler sur plusieurs jours, avant de redevenir disponible dans le domaine public. Si vous souhaitez alors en récupérer l'usage, nous vous conseillons d'utiliser l'outil [Whois d'un nom de domaine](https://www.ovhcloud.com/fr/domains/whois/) pour vérifier sa disponibilité régulièrement.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
>> Au-delà de la période de rédemption, le nom de domaine entre dans une phase de suppression pouvant s'étaler sur plusieurs jours, avant de redevenir disponible dans le domaine public. Si vous souhaitez alors en récupérer l'usage, nous vous conseillons d'utiliser l'outil [Whois d'un nom de domaine](/links/web/domains-whois) pour vérifier sa disponibilité régulièrement.
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 19]
```diff
@@ -0,0 +1,161 @@
...
+- Vous n'avez pas enregistré de moyen de paiement permettant d'effectuer un prélèvement pour le règlement de vos factures.
+
+Il vous suffit alors de choisir l'action `Régler ma facture`{.action} qui vous redirigera vers une plateforme de paiement en ligne.
+
+## FAQ
+
+> [!faq]
+>
+> J'ai oublié de renouveler mon nom de domaine à temps, que faire ?
+>> Commencez par vérifier le statut du nom de domaine en utilisant l'outil [Whois d'un nom de domaine](https://www.ovhcloud.com/fr/domains/whois/).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
>> Commencez par vérifier le statut du nom de domaine en utilisant l'outil [Whois d'un nom de domaine](/links/web/domains-whois).
```

File: pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md [hunk 20]
```diff
@@ -0,0 +1,161 @@
...
+>> S'il s'agit d'un nom de domaine générique (.com, .org, .net, etc.), celui-ci vous reste « réservé » pendant une période dite de « *grâce* » (environ 7 jours) puis pendant celle de « *rédemption* » (environ 35 à 40 jours supplémentaires). Ces délais sont variables et dépendent de l'extension et des règles du registre détenant l'extension. Pendant ces deux périodes, vous seul serez en mesure de renouveler le nom de domaine.
+>> Pendant la période de grâce, vous pourrez le renouveler via l'espace client, en suivant les étapes de ce guide.
+>> Pendant la période de rédemption, le domaine n'est plus géré par OVHcloud. Son renouvellement via votre espace client occasionnera des frais supplémentaires (frais de restauration, qui varient en fonction de l'extension. Voir les tarifs en cliquant sur l'extension concernée [ici](https://www.ovhcloud.com/fr/domains/tld/){.external}.).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/autorenew_domain_name/guide.fr-fr.md]:
```suggestion
>> Pendant la période de rédemption, le domaine n'est plus géré par OVHcloud. Son renouvellement via votre espace client occasionnera des frais supplémentaires (frais de restauration, qui varient en fonction de l'extension. Voir les tarifs en cliquant sur l'extension concernée [ici](/links/web/domains-tld).).
```

**Review by Kazarus4 — COMMENTED** [review]
FRFR typo review done (asked by @Y0Coss

**Comment by Kazarus4** [comment]
Index and meta yaml done. Do not modify this PR and DO NOT MERGE Too. Translations will be start.

**Comment by Y0Coss** [comment]
Thanks for your contribution @fabienOVH

**Pull request merged**